sound/soc/sof/sof-pci-dev.c

```diff
@@ -77,6 +77,13 @@ static const struct dmi_system_id community_key_platforms[] = {
 			DMI_MATCH(DMI_BOARD_NAME, "UP-WHL01"),
 		}
 	},
+	{
+		.ident = "Up Extreme - TGL",
```

Comment:
"Up Xtreme" AFAIK https://up-shop.org/up-xtreme-i11-boards-series.html

Reply:
@lyakh, you are right.
The ident for this should be "UP Xtreme i11"

While at it, we should also fix the WHL version's ident to "UP Xtreme"